spark-doris-connector/src/test/java/org/apache/doris/spark/serialization/TestRowBatch.java

```diff
@@ -57,7 +51,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.google.common.collect.ImmutableList;
+import java.io.ByteArrayOutputStream;
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+import static org.hamcrest.core.StringStartsWith.startsWith;
```

Comment:
startwith not use?

Reply:
It's used, just adjust the order of import